gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -217,9 +216,7 @@ def run(self) -> None:
 
         # create dask cluster
         if self.parsed_args.cluster_config:
-            workers = OmegaConf.load(
-                os.path.join(
-                    self.parsed_args.output_root, "gtsfm", "configs", self.parsed_args.cluster_config))["workers"]
+            workers = OmegaConf.load(self.parsed_args.cluster_config)["workers"]
```

Comment:
Do we need to pass in the full path of the config file?

Reply:
Isnt that a better choice? Using output_root did not make sense at all, that is where outputs are stored. we could use the root folder + "configs/", but maybe its better to just ask the user to provide the full path.